plugins/modules/zos_mount.py

```diff
@@ -1169,6 +1177,7 @@ def main():
             required=False,
         ),
         automove_list=dict(arg_type="str", default="", required=False),
+        tmphlq=dict(type='qualifier_or_empty', required=False, default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.